index.html

```diff
@@ -26,8 +27,82 @@
             </div>
             <a href="#contact" class="navbar__btn">Contact us</a>
         </div>
-
     </nav>
+    <header class="header wrapper">
+        <h1 class="header__title">Sorting Algorithms <br>
+            in Javascript</h1>
+        <button class="header__btn header__btn--color"><a href="#explore"
+                class="header__btn-link header__btn-link--one">Explore</a></button>
+        <button class="header__btn header__btn--signing"> <a href="https://www.google.com/"
+                class="header__btn-link header__btn-link--two"><i class="fa-brands fa-google"></i> Sign up with
+                google</a></button>
+    </header>
+    <main>
+        <section class="website-description wrapper">
+            <div class="website-description__container">
+                <div class="website-description__icons-box">
+                    <div class="website-description__icons-item">
+                        <img src="/img/fire.svg" alt="Fast icon">
+                        <div class="website-description__icons-text">
+                            <h3 class="website-description__icons-header">Fastest possible</h3>
```

Comment:
jest `<h1>` w headerze, jest `<h3>` tutaj, a gdzie `<h2>`? ;)

Reply:
<h2> będzie ale w kolejnej sekcji :) Szczerze mówiąc to <h2> mi tutaj nie pasowało. <h1> jako główny nagłówek, <h3> jako mniej ważne nagłówki  w sekcji "website-description". Jeżeli bym miał nagłówek do całej sekcji to użyłbym właśnie <h2> ;)